test/libsolidity/SyntaxTest.cpp

```diff
@@ -55,41 +55,77 @@ SyntaxTest::SyntaxTest(string const& _filename)
 	m_expectations = parseExpectations(file);
 }
 
-bool SyntaxTest::run(ostream& _stream, string const& _indent)
+bool SyntaxTest::run(ostream& _stream, string const& _linePrefix, int _indent)
 {
 	m_errorList = parseAnalyseAndReturnError(m_source, true, true, true).second;
 	if (!matchesExpectations(m_errorList))
 	{
-		std::string nextIndentLevel = _indent + "\t";
-		_stream << _indent << "Expected result:" << endl;
+		std::string nextIndentLevel = _linePrefix + (_indent < 0 ? string(-_indent, '\t') : string(_indent, ' '));
```

Comment:
This looks rather hacking. Why do you want to use tabs in the first place and especially mix tabs and spaces?